.github/workflows/build.yml

```diff
@@ -229,6 +229,14 @@ jobs:
           FLOW_CONTEXT=Testing ./flow doctrine:migrate --quiet
           bin/phpunit --colors --stop-on-failure -c Build/BuildEssentials/PhpUnit/FunctionalTests.xml --testsuite "Neos tests" --verbose
 
+      - name: Run Parallel Tests for the ES CR
```

Comment:
todo separate step? this takes now 1 minute 30 already while the full suite is 5 min :D That will not scale when when have more tests here :D